Assets/SEE/Game/City/SEECityEvolution.cs

```diff
@@ -70,6 +70,11 @@ public class SEECityEvolution : AbstractSEECity
         [SerializeField, ShowInInspector, Tooltip("The directory in which the GXL files are located."), FoldoutGroup(DataFoldoutGroup), RuntimeTab(DataFoldoutGroup)]
         public DirectoryPath GXLDirectory = new DirectoryPath();
 
+        /// <summary>
+        /// Determines if the graphs should be drawn. If set to true, the graph evolution will be calculated and displayed.
```

Comment:
I still don't have an explanation what problem the field `isDrawn` is supposed to solve. Whatever its intention, it causes other problems because now the evolution city isn't drawn at all. Mind that private fields are not serialized by Unity. 

I removed this field. If there is a convincing reason why it should exist, we can re-add it.